lib/Unexpected.js

```diff
@@ -549,8 +563,14 @@ function calculateAssertionSpecificity(assertion) {
 
 // expect.addAssertion(pattern, handler)
 // expect.addAssertion([pattern, ...], handler)
```

Comment:
The `childUnexpected` parameter is for internal use only, but you're right. I've removed the comment now so the confusion can be avoided.